cmd/kubeadm/app/master/selfhosted_etcd.go

```diff
@@ -164,6 +173,69 @@ func CreateSelfHostedEtcdCluster(cfg *kubeadmapi.MasterConfiguration, client *cl
 	return nil
 }
 
+func createTLSAssets(client *clientset.Clientset) error {
+	if err := createTLSSecret(client, operatorSecret, "etcd", []string{}); err != nil {
+		return err
+	}
+
+	peerDNSNames := []string{
+		fmt.Sprintf("*.%s.%s.svc.cluster.local", etcdCluster, metav1.NamespaceSystem),
+	}
+	if err := createTLSSecret(client, peerSecret, "peer", peerDNSNames); err != nil {
+		return err
+	}
+
+	clientDNSNames := []string{
+		fmt.Sprintf("*.%s.%s.svc.cluster.local", etcdCluster, metav1.NamespaceSystem),
+		fmt.Sprintf("*.%s-client.%s.svc.cluster.local", etcdCluster, metav1.NamespaceSystem),
```

Comment:
What do you mean in the last part? The CA is made available in the secret [here](https://github.com/jamiehannaford/kubernetes/blob/7b77a497b3fef3100b434d1adbac1c39a7964379/cmd/kubeadm/app/master/selfhosted_etcd.go#L228). What do I need to do for the peer's client port?

I updated to v0.3.2 and it's not listening on HTTPS. Logs are [here](https://gist.github.com/jamiehannaford/0eb6ee718f727bf501b97236ed520348).